app/src/main/res/values/strings.xml

```diff
@@ -454,6 +454,10 @@
     <string name="customize_toggle_recent_searches" moz:removedIn="96" tools:ignore="UnusedResources">Recent searches</string>
     <!-- Title for the customize home screen section with Pocket. -->
     <string name="customize_toggle_pocket">Pocket</string>
+    <!-- Title for preference to open wallpapers -->
+    <string name="preferences_wallpapers">Wallpapers</string>
```

Comment:
```suggestion
    <string name="customize_wallpapers">Wallpapers</string>
```

I think we uses `preferences_` for the Settings (one screen before) preferences. Just more naming convention gotchas.

Reply:
Nit away! I'm actually really grateful for such a detailed review. It'll really help me write code that's consistent with what already exists. Also a good indicator that I should maybe be taking a bit more care in general 😅